### PR TITLE
Add fa/analysis command to analyse filings using NLP

### PR DESCRIPTION
**Pull request by DidierRLopes**
Request by a user

### CODE REVIEW

**Comment by jmaslek** [comment]
Haven't tested yet but looks good.

You go between elect.us and eclect.us.

Think all the files should be the latter.

**Review by aia — APPROVED** [review]

**Comment by DidierRLopes** [comment]
> Haven't tested yet but looks good.
> 
> You go between elect.us and eclect.us.
> 
> Think all the files should be the latter.

yea, I'm retard! Thanks for spotting this!

File: gamestonk_terminal/stocks/fundamental_analysis/eclect_us_model.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        Analysis of filings text
+    """
+    result = requests.get(f"https://api.eclect.us/symbol/{ticker.lower()}?page=1")
+
+    if result.status_code == 200:
+        if result.json():
+            return "\n\n".join(
+                {
+                    sentence["sentence"]
+                    for sentence in result.json()[0]["daa_highlights"]
```
Comment (xgboosting) [inline gamestonk_terminal/stocks/fundamental_analysis/eclect_us_model.py:29]: `daa_highlights` is just the discussion and and analysis highlights, there's also an `rf_highlights` section that I find helpful too.

Comment (DidierRLopes) [reply gamestonk_terminal/stocks/fundamental_analysis/eclect_us_model.py:29]: rf is the "risk factors" right?

I'll incorporate that, thanks! :)

Comment (xgboosting) [reply gamestonk_terminal/stocks/fundamental_analysis/eclect_us_model.py:29]: yep 👍

Comment (DidierRLopes) [reply gamestonk_terminal/stocks/fundamental_analysis/eclect_us_model.py:29]: #764 implements this, cheers!

**Pull request merged**